src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -546,6 +547,23 @@ public static Sprite getNumbersSmallCaps()
     return numberSpritesSmallCaps;
   }
 
+  /**
+   * This function returns the sprite sheet for symbol characters that go along with
+   * the letter sprites from getLettersSmallCaps().
+   * Returns a Sprite containing the digits 0-9, one number per frame. The image
```

Comment:
Oh, good, it's not just me who does that.

Reply:
Ah, touché.